Dockerfile.armv6

```diff
@@ -0,0 +1,24 @@
+FROM golang:1.11-stretch AS builder
+
+# Download tools
+RUN wget -O $GOPATH/bin/dep https://github.com/golang/dep/releases/download/v0.5.0/dep-linux-amd64
+RUN chmod +x $GOPATH/bin/dep
+
+# Copy sources
+WORKDIR $GOPATH/src/github.com/pusher/oauth2_proxy
+COPY . .
+
+# Fetch dependencies
+RUN dep ensure --vendor-only
+
+# Build binary
+RUN ./configure && GOARCH=arm GOARM=6 make build
+
+# Copy binary to alpine
+FROM arm32v6/alpine:3.8
+COPY dist/qemu-arm-static /usr/bin/
+RUN apk add --no-cache ca-certificates
+COPY --from=builder /go/src/github.com/pusher/oauth2_proxy/oauth2_proxy /bin/oauth2_proxy
+RUN rm /usr/bin/qemu-*-static -f
```

Comment:
I think this doesn't help much because the space will still be taken in the lower layers of the final docker image.

Is this only needed for `RUN apk add ...`? If so, you could skip that, and just copy `/etc/ssl/certs/ca-certificates.crt` over from the build container.

Reply:
Thanks for looking and good point!
May simplify this a lot. Testing now.